platforms/emulator/runtime/src/components/dma.rs

```diff
@@ -9,14 +9,14 @@ use kernel::component::Component;
 use kernel::create_capability;
 
 pub struct DmaComponent {
-    driver: &'static dma_driver::dma_ctrl::EmulatedDmaCtrl<'static>,
+    driver: &'static dma_driver::axicdma::AxiCDMA<'static>,
```

Comment:
Can we delete the old driver?

Reply:
I saw the comment in the body of the PR that you mentioned you left the old one in. It's okay for now either way.